modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -12,11 +12,16 @@
  * details.
  */
 
+import './DocumentLibrary.scss';
+
 import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
 import ClayCard from '@clayui/card';
 import {ClayInput} from '@clayui/form';
 import ClayIcon from '@clayui/icon';
+import ClayProgressBar from '@clayui/progress-bar';
+import axios from 'axios';
 import {usePage} from 'dynamic-data-mapping-form-renderer';
+import {convertToFormData} from 'dynamic-data-mapping-form-renderer/js/util/fetch.es';
```

Comment:
@alinedoleron, can we add the import following the alphabetic order? This should be on line 22, except if SF tells us to not do it.

Reply:
I realized the order is made according to the alphabetic order after "from".